### ISSUE TITLE
Android - I can claim more than one route in the same turn

### ISSUE THREAD

**Issue by jshammon** [byucs340ta/Spring2019#43]
If I claim a route, then click on the dropdown list again and select a new one, I can claim more than one route in a single turn.

**Comment by jshammon** [comment]
To be fair I think Tanner on my team found this first.

**Comment by bkleinman7** [comment]
...The claim route button disappears when you click claim route on your turn. And our state pattern changes if you successfully claim a route. Not a bug.

**Comment by jshammon** [comment]
Yes the claim route button disappears but then If you click the dropdown and select another route the claim route button comes back and you can claim another route. I was able to do it multiple times.
